src/context/VideoProvider.tsx

```diff
@@ -31,6 +31,9 @@ import { BlendColors } from '../utils/colors';
 import { useVideoDebug } from './useVideoDebug';
 import { VideoContext } from './video';
 
+const DEBUG_PREFIX = 'VideoProvider';
+const log = debug(DEBUG_PREFIX);
```

Comment:
Why not just inline these two lines (as `DEBUG_PREFIX` isn't use in any other context0?